ain_imager_src/qaddcustomobject.cpp

```diff
@@ -141,16 +141,16 @@ void QAddCustomObject::onAddCustomObject() {
 		set_ok(m_dec_line);
 	}
 
-	QRegExp ra_re("\\d*:?\\d*:?\\d*\\.?\\d*");
+	QRegularExpression ra_re("\\d*:?\\d*:?\\d*\\.?\\d*");
 	double ra = indigo_stod(ra_str.toUtf8().data());
-	if (ra < 0 || ra > 24 || !ra_re.exactMatch(ra_str)) {
```

Comment:
match() is not equivalent to exactMxatch()!  i believe it should contain ^ and $ anchors?